### ISSUE TITLE
bug: datetime, empty picker items are read by some screen readers

### ISSUE THREAD

**Issue by shivamsharmanps** [ionic-team/ionic-framework#26809]
### Describe the Bug

Hi,

<ion-datetime></ion-datetime> is not support accessibility, when we can expect to get support, it reads whitespace when use 'md' mode and behave differently and for physically challenge person this will hard, could you bring full accessibility support.

### Expected Behavior

accessibility support for datetime-picker.

### Steps to Reproduce

When clicking on month or year and moving up/down to select then it reads black/space.

### Screenshots

![image](https://user-images.githubusercontent.com/50568306/219111412-8fd407bb-efc8-40c5-aeed-888af6e137e5.png)

### Operating System

Windows 10

### Browser

Chrome/Edge

### Version

Version 110

### Additional Information

Please add accessibility for `<ion-datetime></ion-datetime>.`

**Comment by amandaejohnston** [comment]
Thank you for the details. While I don't have JAWS to test with, we should be able to fix things by adding `aria-hidden="true"` to those elements.

If you are having other accessibility problems as well, I would recommend opening separate issues in the `ionic-framework` repo: https://github.com/ionic-team/ionic-framework/issues/new/choose

**Comment by shivamsharmanps** [comment]
I am looking something like this for web/mobile.
https://www.youtube.com/watch?v=yotebj3ibEE

**Comment by shivamsharmanps** [comment]
> Thank you for the details. While I don't have JAWS to test with, we should be able to fix things by adding `aria-hidden="true"` to those elements.
> 
> If you are having other accessibility problems as well, I would recommend opening separate issues in the `ionic-framework` repo: https://github.com/ionic-team/ionic-framework/issues/new/choose

Actually those element we see when run app ( web ) 
i.e. https://user-images.githubusercontent.com/50568306/219291602-6814f36c-d009-4b66-b220-65f271637c6a.png
so how can I add there and that are inside **shadow-root( open)**

**Comment by amandaejohnston** [comment]
> I am looking something like this for web/mobile. https://www.youtube.com/watch?v=yotebj3ibEE

Please file a separate issue on the `ionic-framework` repo and include a minimal reproduction app so we can troubleshoot further.

> Actually those element we see when run app ( web ) i.e. https://user-images.githubusercontent.com/50568306/219291602-6814f36c-d009-4b66-b220-65f271637c6a.png so how can I add there and that are inside **shadow-root( open)**

We'll need to fix it on our end.
